README.md

```diff
@@ -40,13 +47,13 @@ pip install torchserve torch-model-archiver
     For CPU
 
     ```bash
-    conda create --name torchserve torchserve torch-model-archiver psutil future pytorch sentencepiece torchtext torchvision -c pytorch -c powerai
+    conda create --name torchserve torchserve torch-model-archiver psutil future pytorch torchtext torchvision -c pytorch -c powerai
```

Comment:
we should remove `powerai` channel as well, it is only required for `sentencepiece`.

Reply:
I think basic setup should be stripped down to just `conda create --name torchserve torchserve torch-model-archiver -c pytorch`. It seems to work fine that way for basic usage and doesn't add extra deps in the setup command.

Reply:
Can we please limit our reviews to the section for installing from source? This will help us stay focused in this commit.

This commit only tries to fix what's reported in the issue #312 
For other issues than what this commit is trying to address please feel free to create an issue and we can track them. Thanks.